packages/maskbook/src/plugins/ITO/UI/ITO.tsx

```diff
@@ -564,7 +566,6 @@ function ITO_LoadingFailUI({ retryPoolPayload }: { retryPoolPayload: () => void
 
 export class ITO_LoadingFail extends Component<{ retryPoolPayload: () => Promise<void> }> {
```

Comment:
So the tree should be like right?
```jsx
<ITO_LoadingFail>
    <Suspense>
```

Reply:
Tree is like this since `usePoolPayload()` need to be inside `<Suspense>`
```
<Suspense>
  <ITO_LoadingFail>
    <ITO />
```